api/migrations/20230912181649-updateIssuingAgencyToCode.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+var dbm;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  const mClient = await db.connection.connect(db.connectionString, {
+    native_parser: true,
+  });
+
+  const LegislationActs = {
+    ACT_Env_Management: 'Environmental Management Act',
+    ACT_Int_Pest_Management: 'Integrated Pest Management Act',
+  };
+
+  const agencies = [
+    { agencyCode: 'AGENCY_ALC', agencyName: 'Agricultural Land Commission' },
+    { agencyCode: 'AGENCY_WF', agencyName: 'BC Wildfire Service' },
+    { agencyCode: "AGENCY_ENV_COS", agencyName: 'Conservation Officer Service' },
+    { agencyCode: "AGENCY_EAO", agencyName: 'Environmental Assessment Office' },
+    { agencyCode: "AGENCY_EMLI", agencyName: 'Ministry of Energy Mines and Low Carbon Innovation' },
+    { agencyCode: "AGENCY_ENV", agencyName: 'Ministry of Environment and Climate Change Strategy' },
+    { agencyCode: "AGENCY_ENV_BCPARKS", agencyName: 'BC Parks' },
+    { agencyCode: "AGENCY_OGC", agencyName: 'BC Energy Regulator' },
+    { agencyCode: "AGENCY_ENV_EPD", agencyName: 'Ministry of Environment and Climate Change Strategy' },
+    { agencyCode: "AGENCY_LNG", agencyName: 'LNG Secretariat' },
+    { agencyCode: "AGENCY_AGRI", agencyName: 'Ministry of Agriculture and Food' },
+    { agencyCode: "AGENCY_FLNRO", agencyName: 'Ministry of Forests' },
+    { agencyCode: "AGENCY_FLNR_NRO", agencyName: 'Natural Resource Officers' },
+    { agencyCode: "AGENCY_WLRS", agencyName: 'Ministry of Water, Land and Resource Stewardship' },
+    { agencyCode: "AGENCY_CAS", agencyName: 'Climate Action Secretariat' }
+  ];
```

Comment:
A common constants file in the API could make this less DRY, and easier to maintain

Reply:
since its a migration file, I believe it's run only once and never updated. If there is changes that have to be made in the db a new migration file have to be created.